docs/documentation/features/dsl.md

```diff
@@ -7,6 +7,7 @@ layout: documentation
 # Textual Configuration
 
 Eclipse SmartHome provides the possibility to do fully text-based system setups. This is done using domain specific languages (DSLs) for the different kinds of artifacts.
+In addition, configuration files can be used to configure a lot of system OSGi services.  
```

Comment:
Does it make sense to drop the words "a lot of system"? Can't we used it to configure all OSGi services that could be configured by configuration admin?

Reply:
thats, correct. Thanks.